starter-code/pizza.js

```diff
@@ -1 +1,94 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+
+//intialzing the variables TODO: Transform in an object
+var basis = 10;
+var pepPrice = 1;
+var mushPrice = 1;
+var greenPrice = 1;
+var saucePrice = 3;
+var crustPrice = 5;
+var ingredientsPrice = 0;
+var pizzaPrice = basis + ingredientsPrice;
+
+// Updating the price of the pizza with the right ingredients
+console.log(pizzaPrice);
+ // Callback function to calculate the price
+ function CalculatePrice(){
+  if ($("ul > li:contains(pepperonni)").is(":visible")){
+    ingredientsPrice = ingredientsPrice + pepPrice;
+  }
+  else if ($("ul > li:contains(mushrooms)").is(":visible")){
```

Comment:
You shouldn't use `else if` here, if you have pepperoni on your pizza, every time you call this function the price will be increased by 1

Reply:
I couldn't figure it out another way, that's why I sent a message on slack yesterday to have a hint on what should I start with...